content/en/metrics/overview.md

```diff
@@ -0,0 +1,146 @@
+   - **New users** can use its comprehensive overview of Datadog metric offerings to help increase the value of their metrics.
+   - **Existing users** gain a detailed overview into their current metrics' volumes and origins.
+
+The sections below detail the different parts of the page and provide guidance on how you can maximize the value of your Datadog metrics.
+
+## How your metrics flow through Datadog
+
+{{< img src="metrics/overview/how_metrics_flow.png" alt="The how your metrics flow through Datadog section of the metrics overview page" >}}
+
+The topmost section of the page shows all of your metric sources, along with any configurable processing and a breakdown of the metrics by type (standard or [custom][1]).
+
+### Metric Sources
+
+The **Metric Sources** column shows a summary of the metric sources reporting to Datadog. Click on any of the sources to open the [Summary page][27] scoped to that source. Your Datadog metrics can originate from the following sources:
+
+{{% collapse-content title="Datadog Agents" level="h4" %}}
+The [Datadog Agent][2] collects metrics from the hosts it's installed on, and forwards them to Datadog. These metrics could originate from:
+
+   - Any of the official Datadog integrations that are bundled with the Agent. See [Integration Management][3] for more information about managing these integrations, or the [integrations-core repository][4] for a full listing of the available integrations.
+   - [Custom checks][5], which are used to collect metrics from custom applications or unique systems. You define the logic for the check yourself in the Agent's configuration files. See [Writing a Custom Agent Check][6] for more information.
+   - [DogStatsD][7], a metrics aggregation service bundled with the Datadog Agent. DogStatsD implements the [StatsD][8] protocol with some Datadog-specific extensions.
+   - [Marketplace integrations][9] installed on the Agent. The [Datadog Marketplace][10] is a digital marketplace where Technology Partners can list their paid offerings to Datadog users.
+{{% /collapse-content %}}
+
+{{% collapse-content title="Cloud integrations" level="h4" %}}
+Also known as authentication-based integrations, these integrations are set up in Datadog. You provide the credentials for obtaining metrics, and Datadog makes API calls on your behalf to collect them. Common examples are cloud provider integrations, Slack, and PagerDuty.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Datadog API" level="h4" %}}
+You can send metrics directly to the [Metrics API][11].
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Metrics from other Datadog Services" level="h4" %}}
+Some Datadog services, such as Synthetic Testing, provide standard metrics by default. Other services can be configured to generate custom metrics. This section covers the available options and provides links to relevant documentation.
```

Comment:
Our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#words-and-phrases) requests that we use `service` instead of `product`. With that said, if there's a compelling reason for using `product` instead, I'm fine with that for now. I can follow up with my team later about whether that guideline needs to updated.

Reply:
Reason would be:
1. To align it with copy on our page
2. There are indeed products which allow you to generate metrics.
I'd push here to change this